grakn-engine/src/main/java/ai/grakn/engine/backgroundtasks/distributed/ClusterManager.java

```diff
@@ -70,6 +72,10 @@ public ClusterManager() {
             LOG.debug("Starting Cluster manager on " + ENGINE_ID);
 
             startZookeeperConnection();
+
+            //TODO start state storage, decide using reflection
```

Comment:
Are we still going to be maintaining the in memory and grakn state storage? Do we still need both?